src/extensions/default/bramble/lib/ConsoleManager.js

```diff
@@ -15,8 +17,30 @@ define(function (require, exports, module) {
         var args = data.args;
         var type = data.type || "log";
 
-        // TODO: Show this in Custom Console UI, see issue #1675 in Thimble
+        if (type === "error-handler") {
+            var regex = new RegExp('(blob:.+):([^:]+):(.+)', 'gm'),
+                endingRegex = new RegExp(':([0-9]+):([0-9]+)$', 'gm'),
+                stackTrace = args["stack"].split("@"),
+                newArgs = [];
+            
+            // Handle Blob URLs
+            for(var i = 1; i < stackTrace.length; i++){
+                var stackItem = stackTrace[i].match(regex);
+                stackItem = (BlobUtils.getFilename(stackItem[0].replace(endingRegex, ''))).split('/');
```

Comment:
I don't follow.  The only `/`s I see in that string are within the blob URLs themselves.  Can you clarify what you're a) passing in as your string; b) what you want to get out?  Sorry for being confusing, I'm just trying to understand your goal vs. your method.